kong/db/schema/typedefs.lua

```diff
@@ -573,6 +573,227 @@ typedefs.semantic_version = Schema.define {
   },
 }
 
+local function validate_jwk(key)
+  -- unless it's a reference
+  if kong.vault.is_reference(key) then
+    return true
+  end
+
+  local pk, err = openssl_pkey.new(key, { format = "JWK" })
+  if not pk or err then
+    return false, "could not load JWK"
```

Comment:
Should we return `err` here?
```suggestion
    return false, "could not load JWK " .. (err or "")
```